services/director-v2/tests/unit/with_dbs/test_modules_db_repositories_groups_extra_properties.py

```diff
@@ -116,14 +118,25 @@ async def user(
     return user
 
 
+@pytest.fixture(params=[PRODUCTS_WITH_INTERNET, "other_product"])
```

Comment:
but you do not use anymore this variable ``PRODUCTS_WITH_INTERNET`` in the logic right? I saw the ``s4llite`` hardcoded right?

Reply:
correct. will remove from here